lib/db/schema/patch-039-040.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE IF NOT EXISTS emailBounces (
+  email VARCHAR(255) NOT NULL,
+  bounceType TINYINT UNSIGNED NOT NULL,
+  bounceSubType TINYINT UNSIGNED NOT NULL,
+  createdAt BIGINT UNSIGNED NOT NULL,
+  PRIMARY KEY(email, createdAt)
+) ENGINE=InnoDB;
```

Comment:
Since this is a new table, let's start out on the right foot and use `ENGINE=InnoDB CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci` to ensure full unicode goodness on the email addresses.